app/ts/util/DisplayWarnings.tsx

```diff
@@ -0,0 +1,30 @@
+import Grid from '@material-ui/core/Grid';
```

Comment:
almost 100% identical with displayerrors?

Reply:
almost, yet not quite ;)